src/base/ApplicationDynamoDBTable.ts

```diff
@@ -56,9 +57,11 @@ export class ApplicationDynamoDBTable extends Resource {
       ...config.tableConfig,
       tags: config.tags,
       name: config.prefix,
-      lifecycle: {
-        ignoreChanges: ['read_capacity', 'write_capacity'],
-      },
+      // TODO: lifecycle doesn't appear to be supported by the interface anymore
```

Comment:
@jpetto that's concerning given this [warning on dynamodb in terraform ](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/dynamodb_table )

Perhaps a cdk bug?